3.9/graphs-smart-graphs.md

```diff
@@ -83,159 +75,23 @@ replicated to each participating DB-Server, (weighted) graph traversals and
-     ~var graph_module = require("@arangodb/smart-graph");
-     ~var graph = graph_module._create("myGraph", [], [], {smartGraphAttribute: "region", numberOfShards: 9});
-     ~graph._addVertexCollection("shop");
-     ~graph._addVertexCollection("customer");
-     ~graph._addVertexCollection("pet");
-      var rel = graph_module._relation("isCustomer", ["shop"], ["customer"]);
-      graph._extendEdgeDefinitions(rel);
-      graph_module._graph("myGraph");
-     ~graph_module._drop("myGraph", true);
-    @END_EXAMPLE_ARANGOSH_OUTPUT
-    @endDocuBlock smartGraphCreateGraphHowTo3_cluster
-{% endarangoshexample %}
-{% include arangoshexample.html id=examplevar script=script result=result %}
-
-**Create a Hybrid SmartGraph**
-
-In addition to the attributes you would set to create a SmartGraph, there is an
-additional attribute `satellites` you need to set. It needs to be an array of
-one or more collection names. These names can be used in edge definitions
-(relations) and these collections will be created as SatelliteCollections.
-In this example, both vertex collections are created as SatelliteCollections:
-
-{% arangoshexample examplevar="examplevar" script="script" result="result" %}
-    @startDocuBlockInline hybridSmartGraphCreateGraphHowTo1_cluster
-    @EXAMPLE_ARANGOSH_OUTPUT{hybridSmartGraphCreateGraphHowTo1_cluster}
-      var graph_module = require("@arangodb/smart-graph");
-      var rel = graph_module._relation("isCustomer", "shop", "customer")
-      var graph = graph_module._create("myGraph", [rel], [], {satellites: ["shop", "customer"], smartGraphAttribute: "region", numberOfShards: 9});
-      graph_module._graph("myGraph");
-     ~graph_module._drop("myGraph", true);
-    @END_EXAMPLE_ARANGOSH_OUTPUT
-    @endDocuBlock hybridSmartGraphCreateGraphHowTo1_cluster
-{% endarangoshexample %}
-{% include arangoshexample.html id=examplevar script=script result=result %}
-
-**Create a Hybrid Disjoint SmartGraph**
-
-The option `isDisjoint` needs to be set to `true` in addition to the other
-options for a Hybrid SmartGraph. Only the `shop` vertex collection is created
-as a SatelliteCollection in this example:
-
-{% arangoshexample examplevar="examplevar" script="script" result="result" %}
-    @startDocuBlockInline hybridSmartGraphCreateGraphHowTo2_cluster
-    @EXAMPLE_ARANGOSH_OUTPUT{hybridSmartGraphCreateGraphHowTo2_cluster}
-      var graph_module = require("@arangodb/smart-graph");
-      var rel = graph_module._relation("isCustomer", "shop", "customer")
-      var graph = graph_module._create("myGraph", [rel], [], {satellites: ["shop"], smartGraphAttribute: "region", isDisjoint: true, numberOfShards: 9});
-      graph_module._graph("myGraph");
-     ~graph_module._drop("myGraph", true);
-    @END_EXAMPLE_ARANGOSH_OUTPUT
-    @endDocuBlock hybridSmartGraphCreateGraphHowTo2_cluster
-{% endarangoshexample %}
-{% include arangoshexample.html id=examplevar script=script result=result %}
+(`distributeShardsLike`).
```

Comment:
```suggestion
```
only accept if you take the above comment, I skipped this in the selection before